frontend/src/metabase/query_builder/components/chart-type-selector/ChartTypeSettings/ChartTypeSettings.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from "ttag";
+
+import { ChartTypeList, type ChartTypeListProps } from "../ChartTypeList";
+
+import { OptionLabel } from "./ChartTypeSettings.styled";
+
+type ChartTypeSettingsProps = {
+  sensibleVisualizations: ChartTypeListProps["visualizationList"];
+  nonSensibleVisualizations: ChartTypeListProps["visualizationList"];
+} & Pick<ChartTypeListProps, "selectedVisualization" | "onSelectVisualization">;
+
+export const ChartTypeSettings = ({
```

Comment:
Just curious, are we planning on exposing `ChartTypeSettings` itself in the SDK?

Reply:
Yep!